web/api/APIProvider.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div>
+    <slot />
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, provide, useContext } from "@nuxtjs/composition-api";
+import podAPI from "./v1/pod";
+import nodeAPI from "./v1/node";
+import priorityClassAPI from "./v1/priorityclass";
+import exportAPI from "./v1/export";
+import pvAPI from "./v1/pv";
+import pvcAPI from "./v1/pvc";
+import resetAPI from "./v1/reset";
+import schedulerconfigurationAPI from "./v1/schedulerconfiguration";
+import storageClassAPI from "./v1/storageclass";
+import { PodAPIKey } from "./APIProviderKeys";
+import { NodeAPIKey } from "./APIProviderKeys";
+import { PriorityClassAPIKey } from "./APIProviderKeys";
+import { ExportAPIKey } from "./APIProviderKeys";
+import { PVAPIKey } from "./APIProviderKeys";
+import { PVCAPIKey } from "./APIProviderKeys";
+import { ResetAPIKey } from "./APIProviderKeys";
+import { SchedulerconfigurationAPIKey } from "./APIProviderKeys";
+import { StorageClassAPIKey } from "./APIProviderKeys";
+
+export default defineComponent({
+  setup() {
```

Comment:
> setup() itself does not have access to the component instance 
https://vuejs.org/api/composition-api-setup.html#basic-usage

Therefore, I think that we can't access `$config` and we must use `Plugin` to access its value. 
 (Note: In `Nuxt3`, we maybe use `useRuntimeConfig()` function.)